src/content/docs/tutorial-service-level-mgmt/intro-to-slm.mdx

```diff
@@ -0,0 +1,156 @@
+
+    **Examples SLIs:**
+    * Percentage of login API transactions that complete within 500 milliseconds.
+    * Percentage of login API transactions that complete without an internal server error.
+    * Percentage of login API transactions that complete within 500 milliseconds and without an internal server error.
+    * Percentage of synthetic check Pings against the Login API that are successful.
+    * Percentage of browser logins that transition to the landing page within 3 seconds.
+    * Percentage of metrics ingested that are available for query within 3 seconds.
+    * Some or all of the above aggregated into a single SLI.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-slo"
+    title="Service level objective"
+  >
+    A service level objective (SLO) describes the expected service level of a system, job, or capability. It usuall indicates the required percentage of success of one or more SLIs within a given period of time.
+
+    **Example SLOs:**
+    * The login API will respond within 500 milliseconds and error free 99.99% of the time each week.
+    * The mobile login capability will transition to the landing page within 3 seconds 99.99% of the time each week.
+    * The data ingest of metrics will be available for query within 3 seconds of consumption from the API 99.99% of the time each day.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-service-boundary"
+    title="Service boundary"
+  >
+    A service boundary is the point in a system where the consumer client connects to make requests, also known as the external API or endpoint. Service boundaries can also be internal between two distinct systems, where one collection of applications serves the requests of another collection of applications. For example, an identity management system can serve for both login requests from the client and permissions authorization for different features.
+
+    Types of boundaries:
+    * **Internal service boundary**: Middleware dependencies behind the customer-facing GraphQL API. Each dependency API that responds to the GraphQL tier counts as its own internal service boundary. Internal service boundaries don't represent overall output performance health in a 1:1 ratio. For example, a customer request to the GraphQL API could hit seven internal dependencies. One slow dependency doesn't affect the total sum response time of the original request.
+    * **Customer-facing service boundary**: The customer-facing GraphQL API. This boundary gives you the total sum response time and success metrics required for measuring performance health, without the need to measure each dependency.
+  </Collapser>
+
+  <Collapser
+    id="slm-terminology-error-budget"
+    title="Error budget"
+  >
+An error budget measures and communicates your service level objective compliance. We have more detailed information on this advanced method in our [service level management docs](/docs/service-level-management/intro-slm).
+
```

Comment:
This is great

Reply:
Thank you, this one was a big brain bender for me, I'm glad to hear you liked it!